### PR TITLE
corrects reference to Disneyland, adds related footnote (fixes issue #83)

### PR DESCRIPTION
**Pull request by willbuck**
Disneyworld is where the magic bands are currently deployed, not disneyland. They will be in Disneyland "sometime in 2016" (source: http://www.ocregister.com/articles/disney-646893-mymagic-disneyland.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2607,7 +2607,7 @@ <h2 id="sec-1">Why Are Programmers So Intense About Languages?</h2>
...
+      <p>The industry twists and turns so often, though, that who knows what the next 10 or 20 years will bring? The iPhone, and mobile in general, created a brief renaissance for people who could program using lower-level languages such as Objective-C, people who could worry about a computer&rsquo;s memory. Perhaps the Internet of Things will turn everything into a sensor. (Already you wander Disneyworld with a "magic band" <a id="fnr.40" name="fnr.40" class="footref" href="#fn.40">40</a>, and it watches and tracks you; the whole place is a computer.) This will require yet more low-level thinking. And then there will be websites to make, apps to build, and on and on.</p>
```
Comment (HipsterBrown) [inline index.html]: Minor typo here. Disney World instead of Disneyworld.

File: index.html [hunk 2]
```diff
@@ -2913,6 +2913,12 @@ <h2 id="sec-1">Why Are Programmers So Intense About Languages?</h2>
...
             <p>Actually, programmers don&rsquo;t always talk as much as chat online, often using Slack. Slack is an Internet startup that provides a chat service organized around the needs of geographically diverse groups. A large part of its appeal is its simplicity and the ease with which it integrates with other products. Chat channels can quickly come to represent all the work that&rsquo;s going on inside a company, as a set of streams of text, which can then be searched. Slack provides a short-term institutional memory, which has made a lot of nerds happy. It&rsquo;s growing quickly. Slack&rsquo;s investors recently valued it at $2.6 billion; by the time this issue is published, it will probably be worth more than planet earth.</p>
          </div>
+         <div class="footdef">
+            <sup>
+                <a id="fn.40" name="fn.40" class="footnum" href="#fnr.40">40</a>
+            </sup>
+            <p>A wristband with a built in RF and NFC communicator that acts as your room key, park ticket, and personal identifier. <a href="http://www.wired.com/2015/03/disney-magicband/">Wired did a great piece</a> on the development on the development of the project</p>
```
Comment (HipsterBrown) [inline index.html]: Two more quick suggestions. 

RFID instead of RF.

built-in instead of built in.

File: index.html (hunk 1 above)
Comment (willbuck) [inline index.html]: D'oh, went and fixed that TY!

**Comment by HipsterBrown** [comment]
This looks great. Great add with the footer. :+1:

**Comment by willbuck** [comment]
Thanks for the edit suggestions, made them and yea I'm a Disney nerd and I think it's cool tech so why not share with others!

**Comment by tophtucker** [comment]
Thanks for this @willbuck @HipsterBrown. Closed by https://github.com/BloombergMedia/whatiscode/commit/481d7774c78b5ad9c91ece173bd685d49d79b72b with language by @JimAley.

**Pull request closed without merging**